readthedocs/core/utils/general.py

```diff
@@ -24,8 +21,7 @@ def wipe_version_via_slugs(version_slug, project_slug):
         os.path.join(version.project.doc_path, 'conda', version.slug),
         os.path.join(version.project.doc_path, '.cache'),
     ]
-    for del_dir in del_dirs:
-        broadcast(type='build', task=remove_dirs, args=[(del_dir,)])
+    remove_dirs(del_dirs)
```

Comment:
We might be able to remove the entire `wipe` concept, but that's probably for an additional PR.

Reply:
But this logic doesn't actually work -- we definitely don't want to execute it on the webs, do we?

Reply:
Yeah, I'll look where this can be removed